daemon/graphdriver/windows/windows.go

```diff
@@ -244,7 +244,18 @@ func (d *Driver) Remove(id string) error {
 		return err
 	}
 	os.RemoveAll(filepath.Join(d.info.HomeDir, "sysfile-backups", rID)) // ok to fail
-	return hcsshim.DestroyLayer(d.info, rID)
+
+	layerPath := path.Join(d.info.HomeDir, rID)
+	tmpID := fmt.Sprintf("%s-removing", rID)
+	tmpLayerPath := path.Join(d.info.HomeDir, tmpID)
```

Comment:
How come you are using `path` instead of `filepath`. Won't this create a bunch of `/`?